disable fetching sources.
annoying if you just want to get .spec structure!

it could be parameter to s.parseSpec(), but as prevous code hardcoded, i'll
just "hardcode" different value here.

http://lists.pld-linux.org/mailman/pipermail/pld-devel-en/2015-November/024529.html

one could play with '__urlgetfile' macro, and just tell it 'OK':
%__urlgetfile OK

but then ugly "Fetching" messages are still displayed, which maybe even more confusing.

http://comments.gmane.org/gmane.comp.package-management.rpm.user/159

--- rpm-5.4.16/python/spec-py.c~	2014-10-06 22:44:50.000000000 +0300
+++ rpm-5.4.16/python/spec-py.c	2016-03-16 18:26:35.334750440 +0200
@@ -280,7 +280,7 @@
     char * passPhrase = "";
     char *cookie = NULL;
     int anyarch = 1;
-    int verify = 1;
+    int verify;
     int force = 1;
     char * kwlist[] = {"specfile", NULL};
 
@@ -288,6 +288,8 @@
 				     &specfile))
 	return NULL;
 
+    verify = rpmExpandNumeric("%{?_py_parsespec_verify}%{?!_py_parsespec_verify:0}");
+
     ts = rpmtsCreate();
     if (parseSpec(ts, specfile, "/", recursing, passPhrase,
              	  cookie, anyarch, force, verify) == 0)
